app/uk/gov/hmrc/economiccrimelevyregistration/controllers/GrsContinueController.scala

```diff
@@ -62,6 +66,8 @@ class GrsContinueController @Inject() (
               .flatMap(_ => handleGrsAndBvResult(jd.identifiersMatch, jd.businessVerification, jd.registration))
           }
 
+        case Some(Other) => ???
```

Comment:
I don't think this is needed in the GRS continue controller - GRS should never callback with an 'Other' entity type, since Other doesn't route to GRS. I would update the exception to say 'No valid entity type...'

Reply:
Removed